spec/models/generic_file_spec.rb

```diff
@@ -8,6 +8,16 @@
     @file.apply_depositor_metadata(user.user_key)
   end
 
+  it 'is initialized with blanks' do
+    subject.attributes.each do |k,v|
+      if v.is_a?(Array)
```

Comment:
@mjgiarlo that brings a question on form, should I rebase or make a new commit and then expect someone else to rebase before merging? I don't see that addressed in the guidelines for contributing. Thanks.